features/step_definitions/templates.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const fs = require('mz/fs');
+const request = require('../support/request');
+const path = require('path');
+const Assert = require('chai').assert;
+
+module.exports = function () {
+    this.Given(/^a (valid|invalid)\b job-level template$/, (templateType) => {
+        let targetFile = '';
+
+        switch (templateType) {
+        case 'valid':
+            targetFile = path.resolve(__dirname, '../data/valid-template.yaml');
+            break;
+        case 'invalid':
+            targetFile = path.resolve(__dirname, '../data/invalid-template.yaml');
+            break;
+        default:
+            return Promise.reject('Template type is neither valid or invalid');
+        }
+
+        return fs.readFile(targetFile, 'utf8')
+        .then((contents) => {
+            this.templateContents = contents;
+        });
+    });
+
+    this.When(/^they submit it to the API$/, () =>
+        this.getJwt(this.accessKey)
```

Comment:
u do not need the jwt right, since the enpoint requires no auth?
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/template-validator.js